versioned/spi/src/test/java/org/projectnessie/versioned/TestMetricsVersionStore.java

```diff
@@ -132,7 +132,8 @@ public CommitMeta squash(List<CommitMeta> metadata) {
                         Optional.empty(),
                         CommitMeta.fromMessage("metadata"),
                         Collections.emptyList(),
-                        () -> null),
+                        () -> null,
+                        (k, c) -> {}),
```

Comment:
nit: is this necessary? Could we use the default (old) `commit` method?

Reply:
IIRC that was necessary that time (to use the non-default convenience override)